### DISCUSSION TITLE
Help with proposal

### DISCUSSION THREAD

**Discussion by nashysolutions** [pointfreeco/swift-dependencies#250] [General]
I'm preparing a proposal for my new employer and I believe I once saw a blog post written by the point free team explaining why this library is a great choice for dependency management. I'm struggling to find it. Anyone have a link?

I think the blog post was describing how Apple's implementation influenced their design, which would be key point I would want to use in my pitch.

I've used this library in other commercial projects, so I know it's great. I just need to convince my new boss that Pointfree is the goat 🐐

**Comment by mbrandonw** [comment]
Hi @nashysolutions, the announcement blog post is [here](https://www.pointfree.co/blog/posts/92-a-new-library-to-control-dependencies-and-avoid-letting-them-control-you) and the [docs](https://swiftpackageindex.com/pointfreeco/swift-dependencies/main/documentation/dependencies) have a lot of information too. Not sure if there is something more specific you are looking for.
